taxi/admin.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
```

Comment:
Add blank line between different import groups

use absolute import instead of relative